<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pianxin</class>
 <widget class="QDialog" name="pianxin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="zhijiao">
    <attribute name="title">
     <string>直角坐标法</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_FLWindex">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zj_X">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>方里网索引号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>X坐标：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Y坐标：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zj_Y">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>偏心距离法</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton_left">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_right">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>411</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/pianxin.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pianxin_OK">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pianxin_cancel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>199</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>d1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>d2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>s1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>s2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>A_x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>A_y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>angle:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="d1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="d2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="s1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="s2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="A_x"/>
        </item>
        <item>
         <widget class="QLineEdit" name="A_y"/>
        </item>
        <item>
         <widget class="QLineEdit" name="angle"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>label_6</zorder>
    <zorder>radioButton_left</zorder>
    <zorder>radioButton_right</zorder>
    <zorder>d1</zorder>
    <zorder>d2</zorder>
    <zorder>s1</zorder>
    <zorder>label_3</zorder>
    <zorder>s2</zorder>
    <zorder>A_x</zorder>
    <zorder>A_y</zorder>
    <zorder>angle</zorder>
    <zorder>label_4</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>pianxin_OK</zorder>
    <zorder>pianxin_cancel</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
